tests/functional/BaseCase.php

```diff
@@ -101,17 +106,34 @@ protected function cleanTestProject(): void
         }
 
         $parent = $folderId;
+        $fileStorageBucketName = (string) getenv('BQ_BUCKET_NAME');
         // Iterate over pages of elements
-        $pagedResponse = $projectsClient->listProjects('folders/' . $parent);
+        $pagedResponse = $mainClient->listProjects('folders/' . $parent);
         foreach ($pagedResponse->iteratePages() as $page) {
             /** @var Project $element */
             foreach ($page as $element) {
                 if (str_starts_with($element->getProjectId(), $this->getStackPrefix())) {
-                    $formattedName = $projectsClient->projectName($element->getProjectId());
+                    $formattedName = $mainClient->projectName($element->getProjectId());
                     $billingInfo = new ProjectBillingInfo();
                     $billingInfo->setBillingEnabled(false);
                     $billingClient->updateProjectBillingInfo($formattedName, ['projectBillingInfo' => $billingInfo]);
-                    $operationResponse = $projectsClient->deleteProject($formattedName);
+
+                    $fileStorageBucket = $storageManager->bucket($fileStorageBucketName);
```

Comment:
Nakonec to teda nemazalo len v testoch pretoze tu nepouzvame ten handler pretoze nemame projektove credentials